Fix error message when connecting to elasticsearch 6 or above

Original issue: https://github.com/kuzzleio/documentation/issues/427 by @appacea 

Copy-pasted content:

---

In the following log:

root@openstack-test:~/kuzzle/kuzzle/bin# node kuzzle start
[:information_source:] Starting Kuzzle server
[x] [ERROR] InternalError: _**Your elasticsearch version is 6.1.1; Only elasticsearch version 5.0.0 and above are supported.**_
   at InternalError (/root/kuzzle/kuzzle/node_modules/kuzzle-common-objects/lib/errors/internalError.js:5:5)
   at Bluebird.resolve.client.info.then.response (/root/kuzzle/kuzzle/lib/services/internalEngine/index.js:106:17)
   at process._tickCallback (internal/process/next_tick.js:109:7)

The error message:
Your elasticsearch version is 6.1.1; Only elasticsearch version 5.0.0 and above are supported.

Should read:
Your elasticsearch version is 6.1.1; Only elasticsearch version 5.X is currently supported.

--- a/lib/services/elasticsearch.js
+++ b/lib/services/elasticsearch.js
@@ -81,7 +81,7 @@ class ElasticSearch extends Service {
         this.esVersion = response.version;
 
         if (this.esVersion && !semver.satisfies(this.esVersion.number, '5.x')) {
-          throw new KuzzleInternalError(`Your elasticsearch version is ${this.esVersion.number}; Only elasticsearch version 5.0.0 and above are supported.`);
+          throw new KuzzleInternalError(`Your elasticsearch version is ${this.esVersion.number}; Only elasticsearch version 5 is currently supported.`);
         }
 
         return this;

--- a/lib/services/internalEngine/index.js
+++ b/lib/services/internalEngine/index.js
@@ -118,7 +118,7 @@ class InternalEngine {
         this.esVersion = response.version;
 
         if (this.esVersion && !semver.satisfies(this.esVersion.number, '5.x')) {
-          throw new KuzzleInternalError(`Your elasticsearch version is ${this.esVersion.number}; Only elasticsearch version 5.0.0 and above are supported.`);
+          throw new KuzzleInternalError(`Your elasticsearch version is ${this.esVersion.number}; Only elasticsearch version 5 is currently supported.`);
         }
       }))
       .then(() => this.kuzzle.indexCache.initInternal(this))

